src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -0,0 +1,119 @@
+                                   size='large'>
+                </lightning-spinner>
+                <p class='loading-text slds-text-heading_medium'>{loadingText}</p>
+            </div>
+            <lightning-layout-item size="12">
+                <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
+            </lightning-layout-item>
+            <lightning-layout-item size="12">
+                <!-- Temporary Hardcoded Text until this functionality is moved out of NPSP entirely -->
+                <p class="slds-text-body_regular slds-p-vertical_medium">
+                    NPSP uses the gateway that was added to Elevate
+                    most recently unless you override it. To change the gateway NPSP uses, click Edit, enter the
+                    gateway ID, and save your work. Find the gateway ID in the
+                    Elevate Admin Console | Gateway Management.
+                </p>
+            </lightning-layout-item>
+            <template if:true={isReadOnly}>
+                <lightning-layout-item size="12" class="slds-text-align_center">
+                    <lightning-button
+                            label="Edit"
+                            data-id="editGatewayButton"
+                            onclick={handleEdit}
+                            data-qa-locator="button edit gateway id">
+                    </lightning-button>
+                </lightning-layout-item>
+            </template>
+            <lightning-layout-item size="4">
+                <!-- Temporary Hardcoded Text until this functionality is moved out of NPSP entirely -->
+                <lightning-input variant="label-inline"
+                                 read-only={isReadOnly}
+                                 label="Gateway ID"
+                                 value={gatewayId}
+                                 type="text"
+                                 data-id="gatewayIdField"
+                                 data-qa-locator="input gateway id field">
+                </lightning-input>
+                <template if:false={isReadOnly}>
+                    <lightning-button-group>
+                        <!-- Temporary Hardcoded Text until this functionality is moved out of NPSP entirely -->
+                        <lightning-button
+                                label="Save"
+                                variant="brand"
+                                onclick={handleSave}
+                                data-qa-locator="button gateway management save">
+                        </lightning-button>
+                        <!-- Temporary Hardcoded Text until this functionality is moved out of NPSP entirely -->
+                        <lightning-button label="Cancel"
+                                          onclick={handleCancel}
+                                          data-qa-locator="button gateway management cancel">
```

Comment:
:+1: I think this is ok for QA locators that don't have any dynamic properties going forward.